### ISSUE TITLE
Third enumerate example returns TypeError

### ISSUE THREAD

**Issue by olivertomic** [Sulstice/cocktail-shaker#134]
In the second example the second line has one command too many. `>>> peptide_backbone = PeptideBuilder(2)` should be on the next line.

When running the third example:

>>> from cocktail_shaker import Cocktail
>>> from cocktail_shaker import PeptideBuilder   
>>> peptide_backbone = PeptideBuilder(2)
>>> cocktail = Cocktail(peptide_backbone,ligand_library = ['Br', 'I'])
>>> combinations = cocktail.shake()
>>> enumerations = cocktail.enumerate(dimensionality='2D')

I get the following 

(joss3) C:\Users\olive\Google Drive\Work\joss>python test13.py
Generating 2 Compounds...
Enumerating 2 Compounds....
Traceback (most recent call last):
  File "test13.py", line 7, in <module>
    enumerations = cocktail.enumerate(dimensionality='2D')
  File "C:\Users\olive\anaconda3\envs\joss3\lib\site-packages\cocktail_shaker\functional_group_enumerator.py", line 198, in enumerate
    MoleculeValidator(enumerated_molecules, smiles=True)
  File "C:\Users\olive\anaconda3\envs\joss3\lib\site-packages\cocktail_shaker\validation.py", line 40, in __init__
    self.validate_smiles(self.molecules)
  File "C:\Users\olive\anaconda3\envs\joss3\lib\site-packages\cocktail_shaker\validation.py", line 70, in validate_smiles
    vs(self.molecules[i])
  File "C:\Users\olive\anaconda3\envs\joss3\lib\site-packages\molvs\validate.py", line 119, in validate_smiles
    mol = Chem.MolFromSmiles(smiles)
TypeError: No registered converter was able to produce a C++ rvalue of type class std::basic_string<wchar_t,struct std::char_traits<wchar_t>,class std::allocator<wchar_t> > from this Python object of type Mol

**Comment by Sulstice** [comment]
Huh, this error will get through if my validation functions believe the molecule doesn't actually exist or something is wrong with the 1D String. The error spawns from the RDKit native package not liking a molecule.

**Comment by Sulstice** [comment]
This took some digging but an error on my part that I need to adjust for -> digging deeper it
s actually this piece of code. 

```                elif dimensionality == '2D':
                    if not smiles_enumerated in enumerated_molecules:
                        enumerated_molecules.append(Chem.MolFromSmiles(smiles_enumerated))
``` 

where the molecules are actually RDKit mol objects and not smiles whereas my validator function is expecting smiles.  A patch is being applied

**Comment by Sulstice** [comment]
Patch has been applied in Commit ID: `cce4eeb5d03867499a9f3ce7d5c7cf9829131008 ` on the MASTER branch and now cocktail shaker is released with v1.1.5.
